dotnet/src/Connectors/Connectors.AI.OpenAI/OpenAIModelResultExtensions.cs

```diff
@@ -23,6 +23,16 @@ public static TextModelResult GetOpenAITextResult(this ModelResult resultBase)
         return resultBase.GetResult<TextModelResult>();
     }
 
+    /// <summary>
+    /// Retrieves a typed <see cref="Completions"/> OpenAI / AzureOpenAI result from text completion prompt.
```

Comment:
It seems confusing that this returns the AzureOpenAI result type. Is this what you meant?
```
/// Retrieves a <see cref="StreamingTextModelResult "/> from the model result.
```